router/rootfs/etc/confd/templates/backend.conf

```diff
@@ -0,0 +1,3 @@
+proxy_ssl_verify        off;
```

Comment:
I think this behavior belongs in `nginx.conf` -- without knowing too much about the way the `nginx.conf` file is currently structured, it's usually the place for proxy config, timeouts, etc.